docs/dev/cli/saucectl/imagerunner/artifacts.md

```diff
@@ -0,0 +1,87 @@
+---
+id: artifacts-download
+title: saucectl imagerunner artifacts download
+sidebar_label: artifacts download
+---
+
+Get artifacts/files of the container from a Sauce Orchestrate run. Only the files specified in `artifacts` configuration are downloaded. Supports glob pattern.
+
+## Usage
+
+```bash
+$ saucectl imagerunner artifacts download runID globPattern [flags]
```

Comment:
Easier to read, as it's more clear, especially on such a long command, what is part of the command name itself and what is an argument:
```suggestion
$ saucectl imagerunner artifacts download <runID> <globPattern> [flags]
```